common/constants.go

```diff
@@ -0,0 +1,37 @@
+package common
+
+import (
+	"strconv"
+
+	"github.com/drand/drand/utils"
+)
+
+const (
+	VersionBase = 10
+	VersionSize = 32
+)
+
+var (
+	MAYOR = "0"
+	MINOR = "0"
+	PATCH = "0"
+)
+
+func GetAppVersion() utils.Version {
```

Comment:
consider putting in a `sync.Once` so that this isn't re-run every call?

Reply:
Great advice!! Done!